crates/compiler/can/tests/test_suffixed.rs

```diff
@@ -444,6 +444,26 @@ mod suffixed_tests {
         );
     }
 
+    #[test]
+    fn dbg_expr() {
+        run_test!(
+            r#"
+            main =
+                dbg (dbg 1 + 1)
+            "#
+        );
+    }
+
+    #[test]
+    fn pizza_dbg() {
+        run_test!(
+            r#"
+            main =
+                1 |> dbg
+            "#
+        );
+    }
```

Comment:
These tests are in their own commit because `test_suffixed.rs` is not really an appropriate place for them. All of the other snapshot tests have to do with desugaring the `!` suffix specifically. Would it be better to delete these tests, or generalize this code to support multiple can snapshot test modules?